playbooks/roles/edx_django_service/tasks/main.yml

```
@@ -213,7 +213,9 @@
     - install:app-requirements
 
 - name: "Install Datadog APM requirements"
-  when: COMMON_ENABLE_DATADOG and COMMON_ENABLE_DATADOG_APP
+  when: >
+    COMMON_ENABLE_DATADOG and
+    (COMMON_ENABLE_DATADOG_APP or edx_django_service_datadog_enable)
```

Now that we have the variable with a good default, we can just change this to consult that one variable:

```suggestion
  when: edx_django_service_datadog_enable
```

---

If we instead want to keep consulting the COMMON_ logic here, then the default should just be trimmed down to `edx_django_service_name in COMMON_ENABLE_DATADOG_APP_SERVICES`.

---

duhhh sorry. yes. i was clearly not caffeinated enough this morning.